build/BuildExtension.ps1

```diff
@@ -14,11 +14,19 @@
  # limitations under the License.
  ##>
 [CmdletBinding()]
-Param([string[]]$TasksToBuild, [switch]$SkipInit, [switch]$SkipCompile, [switch]$SkipTest)
+Param(
+    [string[]]$TasksToBuild,
+    [switch]$SkipInit,
+    [switch]$SkipCompile,
+    [switch]$SkipTest,
+    [string]$Publisher,
+    [string]$Version
+)
 
 
 pushd (Join-Path $MyInvocation.MyCommand.Path ..)
-$functionsModule = Import-Module ./BuildFunctions.psm1 -PassThru
+$functionsModule = Import-Module ./BuildFunctions.psm1 -PassThru -Verbose:$false
+$functionsModule.GetVariableFromCallersModule("VerbosePreference").Value = $VerbosePreference
```

Comment:
Just my lack of PS knowledge.  Why do you set this Verbose to false above and change it here? Or are these two diff things?

Reply:
The one above affects the verbosity for the `Import-Module` call, not the resulting module.

This is setting the module scope verbose preference which affects Write-Verbose called by the module functions.